src/diffpy/utils/resampler.py

```diff
@@ -15,25 +15,24 @@
 
 """Various utilities related to data parsing and manipulation."""
 
-import numpy
+import numpy as np
 
 
-# NOTE - this should be faster than resample below and conforms more closely to
-# numpy.interp. I'm keeping resample for legacy reasons.
 def wsinterp(x, xp, fp, left=None, right=None):
     """One-dimensional Whittaker-Shannon interpolation.
 
-    This uses the Whittaker-Shannon interpolation formula to interpolate the value of fp (array),
-    which is defined over xp (array), at x (array or float).
+    Reconstruct a continuous signal from discrete data points by utilizing sinc functions
```

Comment:
We need to keep a reference to Whitaker-Shannon here, preferably a reference to their paper, though I very much like your expanded description so let's keep that.

Reply:
Done - DOI added.